android/src/com/frostwire/android/gui/views/AbstractListAdapter.java

```diff
@@ -504,13 +504,17 @@ protected void initCheckBox(View view, T item) {
     }
 
     protected void initTouchFeedback(View v, T item) {
+        initTouchFeedback(v, item, viewOnClickListener, viewOnLongClickListener, viewOnKeyListener);
+    }
+
+    protected void initTouchFeedback(View v, T item, OnClickListener clickListener, OnLongClickListener longClickListener, OnKeyListener keyListener) {
```

Comment:
more flexible touch feedback initialization method for those list adapters that might behave differently